README.md

```diff
@@ -142,12 +142,17 @@ Starting with an Ubuntu 18.04.5 LTS installation:
 
 * apt install liblingua-en-numbers-ordinate-perl
 
-At this point, should be able to compile tests/hello.tex, but probably
-not much more.
+* cp ./cfg/texml.cfg.template ./cfg/texml.cfg
+
+At this point, should be able to compile, e.g., tests/hello.tex.
+
+### Microsoft Windows 
+
+The above instructions should work in an Ubuntu-based [Windows Subsystem for Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux).
 
 ## Testing
 
-If you install the STIX Two fonts somewhere where fontconfig can find
+If you install the STIX Two fonts and Source Sans Pro somewhere where fontconfig can find
 them, you might also be able to compile test/graphics.tex.  (If you
 want to be able to use stix2.sty (or stix.sty), you'll need the Ubuntu
 fonts-stix package (I think).)
```

Comment:
I was wondering about this line.

Reply:
> I was wondering about this last sentence.

@pkra Sorry, I'm confused.  What is the question?  You addition makes sense.  Is there something else?

Reply:
@davidmjones sorry, I was unclear.

I'm wondering if we want to update (or remove) the following:

>  (If you
want to be able to use stix2.sty (or stix.sty), you'll need the Ubuntu
fonts-stix package (I think).)

But now I had time to take another look. From what I can tell, `fonts-stix` (debian/ubuntu)  installs STIX 1.1. 

I think `tex-live-fonts-extra` includes STIX Two and SourceSans (but comes in at an extra 2GB so perhaps a warning and alternate path should be documented).

Reply:
I've tweaked things some more following our last discussion